include/TPP/Passes.h

```diff
@@ -98,6 +98,8 @@ class XeGPUDialect;
 
 } // namespace mlir
 
+#include "TPP/Conversion/ConvertVectorToXsmm/ConvertTranspose.h"
```

Comment:
We need to make the headers visible to tpp-opt somehow in order to make the passes available in standalone mode. We can include them directly in both tpp-opt and in VectorToXsmm.cpp, or we could include them in Passes.h. I thought it made sense to include them in Passes.h because  they are passes after all, and they ought to be visible wherever the rest of the passes are, but if you think that needn't be, we can do what I said prreviously i.e., include the headers in two places.